clients/src/main/java/org/apache/kafka/clients/consumer/internals/DefaultFetcher.java

```diff
@@ -162,7 +162,7 @@
 
     private CompletedFetch nextInLineFetch = null;
 
-    public Fetcher(LogContext logContext,
+    public DefaultFetcher(LogContext logContext,
```

Comment:
Would it confuse git if we add an empty `Fetcher` interface in this PR? That would make the intent a little easier to understand.

Reply:
OK. Will do.